src/settings.rs

```diff
@@ -496,19 +493,20 @@ impl Store for FileSystemStore {
 
     fn save(&mut self, settings: &Settings) {
         log::info!("Saving new settings to file {}", self.path.display());
-        let display = match settings.fullscreen {
-            true => FULLSCREEN,
-            false => WINDOW,
+        let display = if settings.fullscreen {
+            FULLSCREEN
+        } else {
+            WINDOW
```

Comment:
I actually used the `match` explicitly here because the `if` statement is almost twice as many lines and IMHO actually less clear.